wagtail/images/templates/wagtailimages/images/edit.html

```diff
@@ -108,7 +108,7 @@ <h2 class="label no-float">{% trans "Focal point" %} <span class="w-font-normal"
             </div>
         </div>
 
-        <div class="row row-flush nice-padding u-hidden@sm">
+        <div class="row row-flush nice-padding sm:!w-hidden">
```

Comment:
Nitpick but should these changes be in a separate PR?

Reply:
I thought this would be ok enough to refactor here as well considering those legacy utilities were only used in very few places aside from the sidebar.